### ISSUE TITLE
GMFCS level IV

### ISSUE THREAD

**Issue by jagcsun** [obophenotype/human-phenotype-ontology#7281]
**Preferred term label:**
GMFCS level IV

**Synonyms**
The Gross Motor Function Classification System level IV

**Definition (free text, please give PubMed ID)**
Children with cerebral palsy classified under GMFCS level IV use methods of mobility that require physical assistance or powered mobility in most settings; they may walk for short distances at home with physical assistance or use powered mobility or a body support walker when positioned. At school, outdoors and in the community GMFCS level IV children are transported in a manual wheelchair or use powered mobility (PMID: 27188686).

**Parent term (use hpo.jax.org/app)**
Functional motor deficit HP:0004302

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0002-6670-9157

@pnrobinson

**Comment by pnrobinson** [comment]
As mentioned, I think these terms are not good for HPO because they bundle a number of different items. For HPO, we want to capture the detailed data.
